source/include-link.txt

```diff
@@ -0,0 +1,209 @@
+
+With Driver Installation
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Installing the {+driver-short+} enables using it in a simple and straighforward way. After installing, you can use CMake's ``find_package`` command to integrate the {+driver-short+} with your project. To learn how to install the {+driver-short+}, visit the following guides:
+
+- :ref:`cpp-quick-start-download-and-install` to install from source
+- :ref:`cpp-installation-advanced` to install from packages
+
+To use the ``find_package`` command, create a ``CMakeLists.txt`` file in your project directory. The following example creates a ``CMakeLists.txt`` file in the ``/home/user/project1`` project directory that uses ``find_package``:
+
+.. literalinclude:: /includes/cmake_with_driver_installation.txt
+  :caption: /home/user/project1/CMakeLists.txt
+  :start-after: -- sphinx-include-start --
+
+Then, you can build and run your project. The execution should produce output resembling the following sample output:
+
+.. code-block:: none
+
+   $ cd /home/user/project1
+   $ cmake -Bbuild
+   -- The CXX compiler identification is GNU 14.2.0
+   -- Detecting CXX compiler ABI info
+   -- Detecting CXX compiler ABI info - done
+   -- Check for working CXX compiler: /usr/bin/c++ - skipped
+   -- Detecting CXX compile features
+   -- Detecting CXX compile features - done
+   -- Performing Test CMAKE_HAVE_LIBC_PTHREAD
+   -- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
+   -- Found Threads: TRUE
+   -- Configuring done (0.3s)
+   -- Generating done (0.0s)
+   -- Build files have been written to: /home/user/project1/build
+   $ cmake --build build
+   [ 50%] Building CXX object CMakeFiles/view_and_value.dir/<path-to-mongo-cxx-driver-sources>/examples/bsoncxx/view_and_value.cpp.o
+   [100%] Linking CXX executable view_and_value
+   [100%] Built target view_and_value
+   $ ./build/view_and_value
+   { "team" : "platforms", "id" : { "$oid" : "66f4be6fef9eb8b9240619f0" }, "members" : [ "tyler", "jason", "drew", "sam", "ernie", "john", "mark", "crystal" ] }
+   Got key, key = team
+   Got String!
+   Got key, key = id
+   Got ObjectId!
+   Got key, key = members
+   Got Array!
+   array element: tyler
+   array element: jason
+   array element: drew
+   array element: sam
+   array element: ernie
+   array element: john
+   array element: mark
+   array element: crystal
+   as expected, we have a team
+   document has 3 keys.
+   document keys are:
+   team
+   id
+   members
+
+If you encounter errors, you may need to specify additional options to the initial CMake command. The specific options depend on your particular environment, however there are two options that address most common issues. For instance, if your compiler does not default to at least C++17, use the ``-DCMAKE_CXX_STANDARD=17`` CMake option. If you installed the driver to a non-standard location, you may need to specify the ``-DCMAKE_PREFIX_PATH=/<path-to-mongo-cxx-driver-installation>`` option. For example:
```

Comment:
Since you reference this troubleshooting info later on, I wonder if it should go in a separate "Troubleshooting" section? The Pymongo docs do something similar on a few pages, like [this one](https://www.mongodb.com/docs/languages/python/pymongo-driver/current/connect/stable-api/#troubleshooting). Maybe you could add it as a third subsection under Cmake.

I'd also reword slightly:
```suggestion
If you encounter errors, you may need to specify additional options to the initial CMake command. The specific options depend on your particular environment. However, the following options address most common issues:

- If your compiler does not default to at least C++17, use the ``-DCMAKE_CXX_STANDARD=17`` CMake option.

- If you installed the driver to a non-standard location, specify the ``-DCMAKE_PREFIX_PATH=/<path-to-mongo-cxx-driver-installation>`` option. For example:
```